pkg/blob/nodeserver.go

```diff
@@ -162,7 +173,7 @@ func (d *Driver) NodeStageVolume(ctx context.Context, req *csi.NodeStageVolumeRe
 		}
 	}
 
-	accountName, containerName, authEnv, err := d.GetAuthEnv(ctx, volumeID, protocol, attrib, secrets)
+	accountName, containerName, _, accountKey, err := d.GetAuthEnv(ctx, volumeID, protocol, attrib, secrets)
```

Comment:
is it possible to pass `authEnv` since `accountKey` is not the only way for authentication

Reply:
I have a PR: https://github.com/kubernetes-sigs/blob-csi-driver/pull/349, it looks like we only need `args` and `authEnv` , they are both string arrays